addons/website_slides/controllers/main.py

```diff
@@ -343,6 +343,20 @@ def slides_channel_home(self, **post):
 
     @http.route(['/slides/all', '/slides/all/tag/<string:slug_tags>'], type='http', auth="public", website=True, sitemap=True)
     def slides_channel_all(self, slide_category=None, slug_tags=None, my=False, **post):
+        if slug_tags and request.httprequest.method == 'GET':
```

Comment:
Could you change the commit message so that it is more specific to what is done here?
Thanks 🙂 
Also, try not to use links in commit messages as they could not work one day. This is also applicable for the enterprise PR.

Reply:
And commit message still does not explain why you change this method ?